### PR TITLE
Twitter Feed Added

### PR DESCRIPTION
**Pull request by syedjafer**
Twiiter Feed is been Added to the site .

### CODE REVIEW

**Comment by astronomersiva** [comment]
Hi @syedjafer, Thanks a ton for working on this!

Here are few suggestions:

* The files seem to have been copied to a `website` folder. You may want to move them back to root. That is, `index.html` is now in `website/index.html` in this branch.
*  I guess we can do away with `PySangamam Tweets` in this section. It looks redundant.
screen shot 2018-04-26 at 10 22 19 pm

* I personally feel that the Twitter Roll can be a bit taller. Long tweets will get cut off in this(especially if they have images).
* I really love how you have worked on the selection color. Takes an eye to detail to work on such things :) You might want to consider changing the color for `Py` alone in `PySangamam`. Please refer the image below.
screen shot 2018-04-26 at 10 27 16 pm

* I am not sure if this is happening because of some styles that we are applying but green text on blue background in the "Load More Tweets" button looks a bit odd.
screen shot 2018-04-26 at 10 29 23 pm


Also, I assume the space on the left is for the blog roll? Considering how that might take a few more days and we do not want to delay merging this till then, can you position this normally itself?

Cheers!

File: index.html [hunk 1]
```diff
@@ -163,6 +164,25 @@ <h2 class="text-center"><b>Great! How do I contribute?</b></h2>
...
+      <div class="row no-gutters">
+
+        <div class="offset-1">
+         <!-- HERE COMES THE BLOG  -->
+         <b> Here Comes the Blog</b>
+        </div>
+
+        <div class=" offset-md-8">
+          <h2 class="text-center"><b id="Py">Py</b> <b id="Sangamam">Sangamam</b> Tweets </h2>
+        <a class="twitter-timeline" data-width="500" data-height="300" data-theme="light" data-link-color="#19CF86" href="https://twitter.com/pysangamam?ref_src=twsrc%5Etfw">Tweets by PySangamam </a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (astronomersiva) [inline index.html]: Nitpicking here but this indentation looks a bit off.

File: index.html [hunk 2]
```diff
@@ -163,6 +164,25 @@ <h2 class="text-center"><b>Great! How do I contribute?</b></h2>
...
+         <b> Here Comes the Blog</b>
+        </div>
+
+        <div class=" offset-md-8">
+          <h2 class="text-center"><b id="Py">Py</b> <b id="Sangamam">Sangamam</b> Tweets </h2>
+        <a class="twitter-timeline" data-width="500" data-height="300" data-theme="light" data-link-color="#19CF86" href="https://twitter.com/pysangamam?ref_src=twsrc%5Etfw">Tweets by PySangamam </a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+        </div>
+      </div>
+    </div>
+
```
Comment (astronomersiva) [inline index.html]: Can you please remove all these empty lines?

File: website/CNAME [hunk 3]
```diff
@@ -0,0 +1 @@
+pysangamam.org
```
Comment (astronomersiva) [inline website/CNAME]: It looks like all contents have been moved to `website` folder. Please check this.

**Review by astronomersiva — CHANGES_REQUESTED** [review]

**Comment by astronomersiva** [comment]
Hi @syedjafer , please go through [these comments](https://github.com/PySangamam/website/pull/6#issuecomment-384715347). I can still see the orange-ish selection color for Py in PySangamam. The "Load More Tweets" button can have white text when hovered.

File: index.html [hunk 4]
```diff
@@ -163,6 +164,16 @@ <h2 class="text-center"><b>Great! How do I contribute?</b></h2>
       </div>
     </div>
 
+
+     <div class="container-fluid mar-top-50">
+      <div class="row no-gutters">
+        <div class=" offset-md-3">
+          <h2 class="text-center"><b id="Py">Py</b> <b id="Sangamam">Sangamam</b> Tweets </h2>
```
Comment (astronomersiva) [inline index.html]: We can remove this header. It is redundant.

File: index.html [hunk 5]
```diff
@@ -163,6 +164,16 @@ <h2 class="text-center"><b>Great! How do I contribute?</b></h2>
...
+        <a class="twitter-timeline" data-width="600" data-height="400" data-theme="light" data-link-color="#2B7BB9" href="https://twitter.com/pysangamam?ref_src=twsrc%5Etfw">Tweets by PySangamam </a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (astronomersiva) [inline index.html]: Please remove the `?ref_src=twsrc%5Etfw` from the URL..

File: index.html [hunk 6]
```diff
@@ -174,11 +185,14 @@ <h2 class="text-center"><b>Location</b></h2>
         </div>
       </div>
     </div>
+
```
Comment (astronomersiva) [inline index.html]: I guess we can remove these blank lines too.

**Review by astronomersiva — CHANGES_REQUESTED** [review]

**Comment by astronomersiva** [comment]
Please remove the footer and push. I will merge the Twitter changes now and we can have the footer as a separate PR.

**Comment by astronomersiva** [comment]
@syedjafer sorry for nagging..but there seems to be merge conflicts :/

**Comment by syedjafer** [comment]
It works , http://syedjafer.me/website/

**Review by astronomersiva — APPROVED** [review]

**Comment by astronomersiva** [comment]
Thanks for working on this @syedjafer 🥇

**Comment by syedjafer** [comment]
Its my pleasure @astronomersiva

**Comment by syedjafer** [comment]
About Footer ?

**Pull request merged**